app/Services/Immobilisation/ImmobilisationDocumentService.php

```
@@ -24,11 +28,12 @@ public function generateTransferDocument(AssetTransfer $transfer): string
             data: [
                 'transfer' => $transfer,
             ],
-            filename: 'bon_transport_' . $transfer->id,
+            filename: 'bon_transport_'.$transfer->id,
             type: 'immobilisations',
             position: 'portrait'
```

PHPStan (2.2.7)

[error] Unknown parameter $position in call to method App\Services\Core\DocumentService::generate().


(argument.unknown)